packages/kbn-guided-onboarding/src/components/landing_page/guide_card.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { GuideCard } from './guide_card';
+
+jest.mock('../../kibana_services', () => {
```

Comment:
You'll need to refactor the test mock to import and mock out dependencies.

Reply:
Yes, I haven't completed the refactoring yet. I'm concerned that in the plugin I can't import the code out of the package. There is an error that the module doesn't exist.

Reply:
Oh never mind, I think I figured it out :)